.gitignore

```diff
@@ -4,20 +4,19 @@
 *.pyc
 __pycache__
 db.sqlite3
-migrations
```

Comment:
Are we not ignoring migrations? I don't really know what's best, but just want to know what the reasoning is.

Reply:
We are still. This line got added accidentally to main by someone. What this line would do is ignore the entire migrations folder inside each app. We want to keep the folders and the __init__.py file in each folder. The last few lines of this .gitignore shows how that happens.

Reply:
Didn't realize how big this file was but that makes sense! That was probably me who added that line...

Reply:
Lol, no big deal. I think we just had some miscommunication on Thursday.